src/RpcServer/RelayActor.cs

```diff
@@ -0,0 +1,45 @@
+using Akka.Actor;
+using Neo.Network.P2P.Payloads;
+using System.Collections.Generic;
+using static Neo.Ledger.Blockchain;
+
+namespace Neo.Plugins
+{
+    public class RelayActor : UntypedActor
+    {
+        private readonly NeoSystem neoSystem;
+        private readonly Dictionary<UInt256, IActorRef> senders = new Dictionary<UInt256, IActorRef>();
+
+        public RelayActor(NeoSystem neoSystem)
+        {
+            this.neoSystem = neoSystem;
+            Context.System.EventStream.Subscribe(Self, typeof(RelayResult));
+        }
+
+        protected override void OnReceive(object message)
+        {
+            switch (message)
+            {
+                case IInventory inventory:
+                    {
+                        senders.Add(inventory.Hash, Sender);
+                        neoSystem.Blockchain.Tell(inventory);
+                        break;
+                    }
+                case RelayResult reason:
+                    {
+                        if (senders.Remove(reason.Inventory.Hash, out var actor))
```

Comment:
If we never receive this message, the size of `sender` will increase without limit.

Reply:
Why do we not receive this message?

Reply:
According to the Akka documentation (https://doc.akka.io/docs/akka/current/general/message-delivery-reliability.html)
```
local sends can fail in Akka-specific ways:

if the mailbox does not accept the message (e.g. full BoundedMailbox)
if the receiving actor fails while processing the message or is already terminated
```

Also, this it's a module, so it's easy to forget if we make any change in the core.

Reply:
Any solution.

Reply:
Maybe we can add expire time for this dictionary.